App.js

```diff
@@ -0,0 +1,506 @@
+import React from 'react';
+import {useState, useEffect} from 'react';
+import type {Node} from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  Text,
+  useColorScheme,
+  View,
+  Modal,
+  Image,
+  TouchableOpacity,
+  TextInput,
+  Pressable
+} from 'react-native';
+
+import CustModal from './components/custmodal'
+
+import {
+  Colors,
+  DebugInstructions,
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+import SideMenu from 'react-native-side-menu';
+import GestureRecognizer from 'react-native-swipe-gestures';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import KeepAwake from 'react-native-keep-awake';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import moment from 'moment';
+import {styles} from './styles/Styles';
+
+const Section = ({children, title}): Node => {
+  const isDarkMode = useColorScheme() === 'dark';
+  return (
+    <View style={styles.sectionContainer}>
+      <Text
+        style={[
+          styles.sectionTitle,
+          {
+            color: isDarkMode ? Colors.white : Colors.black,
+          },
+        ]}>
+        {title}
+      </Text>
+      <Text
+        style={[
+          styles.sectionDescription,
+          {
+            color: isDarkMode ? Colors.light : Colors.dark,
+          },
+        ]}>
+        {children}
+      </Text>
+    </View>
+  );
+};
+
+const App: () => Node = () => {
+  const isDarkMode = useColorScheme() === 'dark';
+
+  const backgroundStyle = {
+    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
+  };
+
+  const nmodestyle = {
+    backgroundColor: nMode ? 'black' : 'white',
+  };
+
+  const getsData = async(key) => {
+    try {
+      return await AsyncStorage.getItem(key);
+    } catch (e) {
+      return null
+    }
+  };
+
+  const [isOpen, setisOpen] = useState(false);
+  const [resData, setresData] = useState([]);
+  const [ang, setAng] = useState(1);
+  const [nMode, setnMode] = useState(false);
+  const [canSwipe, setcanSwipe] = useState(false);
+  const [fSize, setfSize] = useState(21);
```

Comment:
Create a separate constant in another file that contains all default values, use like then
`const [fSize, setfSize] = useState(DEFAULT_FONT_SIZE);`